pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -12,60 +12,54 @@
 
 import numpy as np
 
-from pytorch_lightning.overrides.data_parallel import LightningDistributedDataParallel
-
 
 class Callback(object):
-    r"""Abstract base class used to build new callbacks.
-    """
+    """Abstract base class used to build new callbacks."""
```

Comment:
I am not familiar with all this `ABC` stuff, so I don't really know :)
But anyway I think it would be better to do it in another PR (if it really should be done)

Reply:
fyi: https://www.python-course.eu/python3_abstract_classes.php